src/imageupload/imageuploadediting.js

```diff
@@ -101,36 +101,25 @@ export default class ImageUploadEditing extends Plugin {
 			this.listenTo( editor.plugins.get( 'Clipboard' ), 'inputTransformation', ( evt, data ) => {
 				const fetchableImages = Array.from( editor.editing.view.createRangeIn( data.content ) )
 					.filter( value => isLocalImage( value.item ) && !value.item.getAttribute( 'uploadProcessed' ) )
-					.map( value => fetchLocalImage( value.item ) );
+					.map( value => { return { promise: fetchLocalImage( value.item ), imageElement: value.item }; } );
 
 				if ( !fetchableImages.length ) {
 					return;
 				}
 
-				evt.stop();
+				const writer = new UpcastWriter();
```

Comment:
Hmm I'm worried about that construct. I think that we could expose those methods on the view if they're need. Then we could write it as:

```js
const view = editor.editing.view; // ps.: also used for crateRangeIn()

/// the code as now

view.setAttribute( 'src', '', fetchableImage.imageElement );
```
This unfortunately will require small changes in the engine.